src/features/dappLogin/DappLoginScreen.tsx

```diff
@@ -26,6 +23,39 @@ import {
   RootNavigationProp,
   RootStackParamList,
 } from '../../navigation/rootTypes'
+import { getKeypair } from '../../storage/secureStorage'
+
+// TODO: Verify crowdspot still works
```

Comment:
We should test this with crowdspot before merging

Reply:
Tested. It works.